openedx/features/ucsd_features/ecommerce/EcommerceClient.py

```diff
@@ -0,0 +1,43 @@
+"""
+Custom client to communicate with ecommerce service
+"""
+import logging
+
+from openedx.core.djangoapps.commerce.utils import ecommerce_api_client
+
+
+logger = logging.getLogger(__name__)
+
+
+class EcommerceRestAPIClient:
+    def __init__(self, user, session=None):
+        self.client = ecommerce_api_client(user, session)
+
+    def assing_voucher_to_user(self, user, course_key):
```

Comment:
I think it would be better if we create a variable like `self.user = user` instead of passing it in functions